PostHog vs LogRocket

joethreepwood opened the pull request:
## Changes

LogRocket comparison

Still waiting on art from @lottiecoxon via #5450 but copy is good for review now

Followed the framework set up in other comparisons, but deviated where I felt it was useful for brevity (or, in the case of apps, to avoid requiring substantive and constant updates). 

## Checklist
- [ ] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [ ] Feature names are in **[sentence case too]([https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case))**
- [ ] Words are spelled using American English
- [ ] I have checked out our [style guide](https://github.com/PostHog/posthog.com/blob/master/STYLEGUIDE.md)

andyvan-ph commented:
Looks like something went wrong on this as it's the Mixpanel comp. with a different file name.

joethreepwood commented:
Yep, my bad - I forgot to hit save on the changes. Just pushed an update. 🤦

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,775 @@
+
+### Library support
+
+<div className="overflow-x-auto -mx-5 px-5">
```
This table is borked

ivanagas replied:
I tried fixing it, but couldn't get it to line up well. The best option (if you want to solve it) seems to be adding another column like the [posthog vs mixpanel post](https://posthog.com/blog/posthog-vs-mixpanel#client-libraries)

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,775 @@
+        </tr>
+    </tbody>
+</table>
+</div>
```
This table could really do with some categorization. It's a bit large and hard to parse atm.

andyvan-ph replied:
Just to repeat the above, but this table is way to big to not have categories / some organisation – i.e. have a section for CDPs, a section for marketing integrations etc. It doesn't have to separate tables to do this, though.

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,775 @@
+
+Some examples of PostHog’s site apps include:
+
+**[User Notifications](/apps/notification-bar):** Display a site banner to alert users about important information.
+**[Schedule interviews](/apps/user-interviews):** Target different types of users to receive Calendly notification, so you can arrange feedback calls. 
+**[In-app prompts](/apps/feedback-widget):** Display in-app prompts which ask for written feedback, or bug reports. 
+**[Raining Pineapples](/apps/pineapple-mode):** Sometimes you just want it to rain Pineapples. You’re welcome. 
```
```suggestion
- **[User Notifications](/apps/notification-bar):** Display a site banner to alert users about important information.

- **[Schedule interviews](/apps/user-interviews):** Target different types of users to receive Calendly notification, so you can arrange feedback calls. 

- **[In-app prompts](/apps/feedback-widget):** Display in-app prompts which ask for written feedback, or bug reports. 

- **[Raining Pineapples](/apps/pineapple-mode):** Sometimes you just want it to rain Pineapples. You’re welcome. 
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,775 @@
+        </tr>
+    </tbody>
+</table>
+</div>
```
This table is broken.

Also, does LogRocket support webhooks? If so, Slack etc. will be a thing.

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,775 @@
+        </tr>
+    </tbody>
+</table>
+</div>
```
Table broken. Also, says Mixpanel.

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,775 @@
+
+## Frequently asked questions
+
+How much does LogRocket cost?
+How much does PostHog cost?
+How do LogRocket and PostHog compare?
+Do LogRocket and PostHog offer free trials?
+Is PostHog or LogRocket more popular?
```
```suggestion
- How much does LogRocket cost?
- How much does PostHog cost?
- How do LogRocket and PostHog compare?
- Do LogRocket and PostHog offer free trials?
- Is PostHog or LogRocket more popular?
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,775 @@
+
+PostHog has transparent pricing which is based only around the amount of usage. It’s free to get started and completely free for the first 1 million events and 15,000 sessions captured every month.
+
+After this free monthly allowance, PostHog charges $0.00045/event and $0.005/recording, Volume, non-profit and [startup discounts](/startups) are available upon request, and we recommend trying [our pricing calculator](/pricing) to estimate your pricing. 
```
```suggestion
After this free monthly allowance, PostHog charges $0.00045/event and $0.005/recording. Volume, non-profit and [startup discounts](/startups) are available upon request, and we recommend trying [our pricing calculator](/pricing) to estimate your pricing. 
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,775 @@
+
+After this free monthly allowance, PostHog charges $0.00045/event and $0.005/recording, Volume, non-profit and [startup discounts](/startups) are available upon request, and we recommend trying [our pricing calculator](/pricing) to estimate your pricing. 
+
+### How do LogRocket and PostHog compare?
+
+PostHog and LogRocket contain some similar features, but are ultimately intended for very different needs and types of users. 
+
+LogRocket is built primarily for front-end developers who need to monitor performance and gather usage information. It has robust session recording and error monitoring tools to accomplish this. 
+
+PostHog is built for engineers and developers in a more general capacity and offers a wider range of tools, which are also suitable for product and growth teams. It also offers robust session recording tools and the ability to track errors, but also includes feature flagging, experimentation and more. 
```
This feels a bit unnecessary, surely?

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,775 @@
+
+With PostHog, it’s free to get started and all users get their first 1 million events and 15,000 sessions for free, every month. Billing limits can be used to keep usage beneath this allowance, enabling you to use PostHog for free indefinitely. 
+
+### Is PostHog or LogRocket more popular?
```
I don't really like the "popularity" framing – feels a bit superficial. Surely just to ask "Who uses PostHog? / Who uses LogRocket?" You can cover the same stuff to add some social proof.

andyvan-ph reviewed (commented):
Done a first pass. A few of the tables are looking a bit janked.

joethreepwood commented:
Should have addressed all that feedback. Having trouble previewing this, as mentioned, but the tables should work now. 

Just needs an image.

pjhul commented:
Looks like there was just one extra `</tr>` that was causing the build to fail ✅

joethreepwood commented:
Ready for review then, @andyvan-ph

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+
+-   LogRocket focuses heavily on helping frontend developers to detect and solve issues. It offers tools such as session recording, analytics and performance monitoring.
+
+-   PostHog is an all-in-one platform that helps teams build better products. It includes many of the same tools as LogRocket, including session recording, analytics and performance monitoring — but also extends to feature flags, A/B experiments and more.
```
```suggestion
-   PostHog is an all-in-one platform that helps teams build better products. It includes many of the same tools as LogRocket, including session recording, analytics and performance monitoring — but also extends to feature flags, A/B testing and more.
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+
+-   PostHog is an all-in-one platform that helps teams build better products. It includes many of the same tools as LogRocket, including session recording, analytics and performance monitoring — but also extends to feature flags, A/B experiments and more.
+
+In this article we’ll dive deeper into these differences and will:
```
```suggestion
In this article, we will:
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+
+### 2. PostHog is open source and transparent
+
+PostHog is built with transparency at its core. Not only do we work in the open and gives full access to [our source code](https://github.com/PostHog/posthog), we also enable others to [build integrations](/docs/apps/build) or [other services on top of PostHog](/how-we-built-an-app-server) and to raise their own PRs, or feedback on [our roadmap](/roadmap).
+
+This approach offers several advantages that closed-source products such as LogRocket can’t offer. For example, if you have internal tools you need to integrate then PostHog’s app framework makes that a breeze — and you can always find out exactly what we’re working on.
```
```suggestion
PostHog is built with transparency at its core. Not only do we work in the open and give full access to [our source code](https://github.com/PostHog/posthog), we also enable others to [build integrations](/docs/apps/build) or [other services on top of PostHog](/how-we-built-an-app-server), raise their own PRs, or feed back on [our roadmap](/roadmap). PostHog's open app framework makes it easy to integrate internal tools, an advantage closed-source products like LogRocket can't offer.
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+
+### 3. PostHog constantly ships new features
+
+At PostHog, we ship new features _constantly_. Every week we update [our changelog](/blog/posthog-changelog) with a recap of what’s launched in the last few days, but often there’s even more that’s being tested in beta. And that’s without counting all the apps and integrations submitted by our community!
+
+In short, we work hard to keep PostHog on the cutting edge and we’re transparent about the ways we do that. We ship weirdly fast.
```
```suggestion
At PostHog, we ship weirdly fast. We update [our changelog](/blog/posthog-changelog) with a recap of new features every week, and often there’s even more in beta testing. And that’s without counting all the apps and integrations submitted by our community! We work hard to keep PostHog on the cutting edge and we’re transparent about the ways we do that.
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+
+### 1. PostHog does more than frontend monitoring
+
+LogRocket is a frontend performance monitoring platform, but PostHog goes well beyond that to include features which are useful in a greater number of roles and teams. PostHog offers all of the major features available in LogRocket, such as session recording and product analytics, as well as tools such as feature flags, A/B experimentation and SQL querying.
+
+LogRocket’s focus on frontend issues means it does offer a few specialist features which PostHog doesn’t currently replicate. However, for the vast majority of teams and organizations, PostHog offers greater value overall and is more widely useful.
```
```suggestion
PostHog goes beyond frontend monitoring by augmenting product analytics and session recording with powerful product and data tools, such as feature flags, A/B testing, and SQL querying. While LogRocket’s narrow focus on frontend issues means it offers a few specialist features, PostHog offers greater value and is more widely useful for the vast majority of teams and organizations
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+-   Discuss what integrations are available with other tools
+-   Explore other frequently asked questions about PostHog and LogRocket
+
+## How do PostHog and LogRocket differ?
```
Feel free to tweak, but I like to get these sections to tight single paragraphs.

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+
+## PostHog and LogRocket feature comparison
+
+PostHog offers a much wider selection of tools than LogRocket, including several tools which will be helpful to frontend developers — such as feature flags, and experimentation. PostHog is also entirely open source, meaning that it offers far greater extensibility than LogRocket.
```
```suggestion
PostHog offers a much wider selection of tools than LogRocket, including several tools that are helpful to frontend developers — such as feature flags, and experimentation. PostHog is also entirely open source, meaning that it offers far greater extensibility than LogRocket.
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+
+-   **Feature flags:** PostHog offers robust, multivariate feature flags which support JSON payloads. This enables you to push real-time changes to your product without needing to redeploy. Visit [our feature flag page](/product/feature-flags) for more information. LogRocket doesn’t have any in-built feature flag functions.
+
+-   **Experiments:** PostHog offers multivariate experimentation, which enable you to test changes and find statistically relevant insights. Visit [the experimentation page](/product/experimentation-suite) for more information. LogRocket doesn’t have any in-built experimentation features.
```
```suggestion
-   **Experiments:** PostHog offers multivariate experimentation, which enables you to test changes and discover statistically relevant insights. Visit [the experimentation page](/product/experimentation-suite) for more information. LogRocket doesn’t have any in-built experimentation features.
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+
+### Product analytics
+
+While LogRocket contains some product analytics tools, it isn’t primarily intended as a analytics platform and lacks many features that product managers and engineers may require. PostHog is a more capable product analytics platform, offering advanced features such as correlation analysis.
```
```suggestion
While LogRocket contains some product analytics features, it isn’t primarily intended as an analytics platform and lacks many features that product managers and engineers may require. PostHog is a more capable product analytics platform, offering advanced features such as correlation analysis.
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+
+-   **Correlation analysis:** This feature enables you to [automatically find correlated events or properties](/manual/correlation) which affect the conversion rate of users within a funnel. LogRocket doesn’t offer any such automated correlation discovery, meaning users must search for correlating factors manually and without assistance.
+
+-   **Query editor:** LogRocket doesn’t currently support any way for users to interrogate data directly, either through SQL or other, abstracted query language. PostHog also doesn’t offer SQL queries…but [we’re actively working on it](/roadmap).
```
```suggestion
-   **Query editor:** LogRocket doesn’t currently support any way for users to interrogate data directly, either through SQL or other, abstracted query language. PostHog also doesn’t offer SQL queries... but [we’re actively working on it](/roadmap).
```

I guess you're using Google Docs? Can you turn-off auto ellipsis? It renders weird online.

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+
+### Session recording
+
+LogRocket and PostHog both offer robust, full-featured session recording tools which outperform even dedicated tools, such as HotJar. LogRocket’s session recording offers a handful of extra features, such as mobile recording, though PostHog is actively working on developing these.
```
```suggestion
LogRocket and PostHog both offer robust, full-featured session recording tools that outperform even dedicated tools, such as HotJar. LogRocket’s session recording offers a handful of extra features, such as mobile recording, though PostHog is actively working on developing these.
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+            <td><strong>Mobile app recordings</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Capture recordings from mobile devices</td>
```
```suggestion
            <td>Capture recordings in iOS and Android apps</td>
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+</table>
+</div>
+
+**Toolbar:** PostHog’s heatmap functionality (technically, clickmapping) is accessed via the [toolbar](/manual/toolbar). It enables you to visualize clicks as an overlay on a live page, but also enables you to quickly create events and actions which can be analyzed using PostHog’s analytics features.
```
Not sure we should add the "techincally clickmapping" bit here. Feels a bit jarring.

joethreepwood replied:
I'll defer to you and you can commit in a change if you want it. But I think it's important to be clear because what we call heatmapping in PostHog isn't actually heatmapping at all.

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+
+### Site apps
+
+PostHog’s extensions also include [site apps](/tutorials/build-site-app), which enable you to inject arbitrary code into your website — something which isn’t possible with LogRocket.
```
```suggestion
PostHog’s extensions also include [site apps](/tutorials/build-site-app), which enable you to inject arbitrary code into your website — something that isn’t possible with LogRocket.
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+
+### How much does PostHog cost?
+
+PostHog has transparent pricing which is based only around the amount of usage. It’s free to get started and completely free for the first 1 million events and 15,000 sessions captured every month.
```
```suggestion
PostHog has transparent pricing based on usage. It’s free to get started and completely free for the first 1 million events and 15,000 sessions captured every month.
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,750 @@
+
+PostHog has transparent pricing which is based only around the amount of usage. It’s free to get started and completely free for the first 1 million events and 15,000 sessions captured every month.
+
+After this free monthly allowance, PostHog charges $0.00045/event and $0.005/recording. Volume, non-profit and [startup discounts](/startups) are available upon request, and we recommend trying [our pricing calculator](/pricing) to estimate your pricing.
```
```suggestion
After this free monthly allowance, PostHog charges $0.00045/event and $0.005/recording and charges progressively less the more your use. Volume, non-profit and [startup discounts](/startups) are available upon request, and we recommend trying [our pricing calculator](/pricing) to estimate your pricing.
```

andyvan-ph reviewed (commented):
Left some suggestions. The only thing blocking merging this is that mega table of integrations, which could do with some categorization.

joethreepwood commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,744 @@
+---
+date: 2023-03-03
+title: PostHog vs Logrocket
```
```suggestion
title: PostHog vs LogRocket
```

joethreepwood commented:
Fixed the image and split the table into two. I'm against trying to list all integrations, so I just lumped them into two categories. All other suggestions merged in, so happy for you to merge when you're ready @andyvan-ph

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,768 @@
+            <td className="w-3/12"></td>
+            <td><strong>LogRocket</strong></td>
+            <td><strong>PostHog</strong></td>
+            <td></td>
```
```suggestion
```

andyvan-ph commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,768 @@
+
+### Popular integrations 
+
+Below, we've listed a few of the most popualr integrations used across PostHog and LogRocket. 
```
```suggestion
Below, we've listed a few of the most popular integrations used across PostHog and LogRocket. 
```

andyvan-ph reviewed (commented):
Yeah, that table looks way better. Changed the first one to "Data pipelines" as it isn't exclusively warehouses in there.

@ivanagas, would you mind give this a final once over just to make sure Joe and I haven't missed anything?

ivanagas commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,767 @@
+
+-   **Correlation analysis:** This feature enables you to [automatically find correlated events or properties](/manual/correlation) which affect the conversion rate of users within a funnel. LogRocket doesn’t offer any such automated correlation discovery, meaning users must search for correlating factors manually and without assistance.
+
+-   **Query editor:** LogRocket doesn’t currently support any way for users to interrogate data directly, either through SQL or other, abstracted query language. PostHog also doesn’t offer SQL queries... but [we’re actively working on it](/roadmap).
```
```suggestion
-   **Query editor:** LogRocket doesn’t currently support any way for users to interrogate data directly, either through SQL or other, abstracted query language. PostHog also doesn’t offer SQL queries... but [we’re actively working on it](/manual/hogql).
```

ivanagas commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,767 @@
+
+### Session recording
+
+LogRocket and PostHog both offer robust, full-featured session recording tools that outperform even dedicated tools, such as HotJar. LogRocket’s session recording offers a handful of extra features, such as mobile recording, though PostHog is actively working on developing these.
```
```suggestion
LogRocket and PostHog both offer robust, full-featured session recording tools that outperform even dedicated tools, such as [Hotjar](/blog/best-open-source-session-replay-tools). LogRocket’s session recording offers a handful of extra features, such as mobile recording, though PostHog is actively working on developing these.
```

ivanagas commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,767 @@
+
+Both PostHog and LogRocket are capable of integrating with third-party tools in a variety of ways. This enables you to accomplish tasks such as exporting data to another platform, importing data for analysis, or enabling two-way communication between the tools.
+
+[PostHog offers more than 50 integrations and apps](/apps), while [LogRocket offers slightly less](https://logrocket.com/features/integrations).
```
```suggestion
[PostHog offers more than 50 integrations and apps](/apps), while [LogRocket offers slightly fewer](https://logrocket.com/features/integrations).
```

ivanagas commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,767 @@
+
+### Site apps
+
+PostHog’s extensions also include [site apps](/tutorials/build-site-app), which enable you to inject arbitrary code into your website — something that isn’t possible with LogRocket.
```
```suggestion
PostHog’s extensions also include [site apps](/tutorials/build-site-app), which enable you to easily add frontend components to your site — something that isn’t possible with LogRocket.
```

ivanagas replied:
inject arbitrary code sounds needlessly aggressive 🧑‍💻

ivanagas commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,767 @@
+</table>
+</div>
+
+> PostHog is not currently SOC 2 certified, but is currently being audited in order to complete this certification.
```
```suggestion
> PostHog is not currently SOC 2 certified, but is [currently being audited](/handbook/company/security) in order to complete this certification.
```
I assume this will update and keep people up to date if they care

joethreepwood replied:
Yeah, when we clear SOC 2 we should do a sweep and correct all sources. @andyvan-ph

ivanagas reviewed (approved):
Some minor points and fixes, but looks good to me 👍

joethreepwood commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,767 @@
+
+## PostHog and LogRocket feature comparison
+
+PostHog offers a much wider selection of tools than LogRocket, including several tools that are helpful to frontend developers — such as feature flags, and experimentation. PostHog is also entirely open source, meaning that it offers far greater extensibility than LogRocket.
```
```suggestion
PostHog offers a much wider selection of tools than LogRocket, including several tools that are helpful to frontend developers — such as feature flags, and experimentation. PostHog is also entirely open source, meaning that it offers far greater transparency than LogRocket. You can even develop your own PostHog apps!
```

joethreepwood commented on contents/blog/posthog-vs-logrocket.md:
```diff
@@ -0,0 +1,766 @@
+---
+date: 2023-03-14
```
```suggestion
date: 2023-03-15
```
